src/main/scala/beam/sim/BeamServices.scala

```diff
@@ -113,8 +114,10 @@ object BeamServices {
   }
 
   def getTazTreeMap(file: String): TAZTreeMap = {
-    Try(TAZTreeMap.fromCsv(file)).getOrElse {
-      BeamServices.defaultTazTreeMap
+    try {
```

Comment:
I see no point in wrapping this in a Try at all if we want the exception to bubble up. This seems as though it can just be the expression if the try and default are indeed not wanted.